### PR TITLE
Add Tile component and story

### PR DESCRIPTION
**Pull request by TonyGorez**

### CODE REVIEW

File: src/components/Tile.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+const Tile = ({
+  children,
+  type,
+  padding
+}) => {
+  const flexClasses = classnames({
```
Comment (G-Ray) [inline src/components/Tile.jsx]: you mean `classes` instead, right ?

File: src/components/Tile.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  return (
+    <div className={flexClasses}>
+      {children}
+    </div>
+  )
+}
+
+Tile.propTypes = {
+  children: PropTypes.oneOfType([
```
Comment (G-Ray) [inline src/components/Tile.jsx]: Do you have some documentation about his ? I usually set the children to be a `func` IIRC (If I Rembember Correctly)

File: stories/tile.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+storiesOf('Tile', module)
+  .add('Default', () => (
+    <Tile
+      children='Default'
+      type='default'
+    />
+  ))
+  .add('Muted', () => (
+    <Tile
```
Comment (G-Ray) [inline stories/tile.js]: I think we should let the user choose what to use as children, for instance from the doc:
```html
<div class="uk-tile uk-tile-secondary">
   <p class="uk-h4">Secondary</p>
</div>
```

File: src/components/Tile.jsx (hunk 2 above)
Comment (TonyGorez) [reply src/components/Tile.jsx]: `PropTypes.element` in the doc !

**Pull request merged**